discord.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <meta charset="utf-8">
+  <link rel="canonical" href="https://discord.gg/KfpqwZB">
+  <script>location="https://discord.gg/KfpqwZB"</script>
+  <meta http-equiv="refresh" content="0; url=https://discord.gg/KfpqwZB">
+  <meta name="robots" content="noindex">
+  <h1>KAMI Blue Discord</h1>
+  <a href="https://discord.gg/KfpqwZB">Click here if you are not redirected.</a>
+    <meta content="https://kamiblue.org/favicon.png" property="og:image">
+    <meta content="KAMI Blue" property="og:title">
+    <meta content="Join the KAMI Blue Discord: https://discord.gg/KfpqwZB" property="og:description">
+    <meta content="Join the KAMI Blue Discord: https://discord.gg/KfpqwZB" name="description">
+    <meta name="theme-color" content="#36393E"> <!-- Normally, the KAMI Blue website colour is "9b90ff". However I set it to the Discord Dark Theme background color so it looks cool and swag and hip and dope -->
+    <link href="https://discord.gg/KfpqwZB" rel="canonical">
+    <meta content="https://discord.gg/KfpqwZB" property="og:url">
+    <meta content="KAMI Blue" property="og:site_name">
+    <link href="https://kamiblue.org/favicon.png" id="favicon" rel="icon" type="image/x-icon">
```

Comment:
You can replace the url to the favicon with `/favicon.png` I'm pretty sure

Reply:
Yeah I'll do that.

Reply:
Nevermind, it doesn't work, /backdoored doesn't use it